src/gConsent/request/issueAccessRequest.test.ts

```diff
@@ -312,6 +314,66 @@ describe("issueAccessRequest", () => {
       expect.anything()
     );
   });
+
+  it("includes the inherit flag if set to false", async () => {
```

Comment:
since inherit is optional should we also have a testcase where it is not included on the issueAccessRequest?

Reply:
Good point, now fixed in 9338aff493481496d2299c9160452ca3277322b7